debezium-testing/debezium-testing-system/src/test/java/io/debezium/testing/system/tests/mysql/MysqlTestsWithDbOperations.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.testing.system.tests.mysql;
+
+import static io.debezium.testing.system.tools.ConfigProperties.DATABASE_MYSQL_PASSWORD;
+import static io.debezium.testing.system.tools.ConfigProperties.DATABASE_MYSQL_USERNAME;
+
+import java.sql.SQLException;
+
+import io.debezium.testing.system.assertions.KafkaAssertions;
+import io.debezium.testing.system.tests.ConnectorTest;
+import io.debezium.testing.system.tools.databases.SqlDatabaseClient;
+import io.debezium.testing.system.tools.databases.SqlDatabaseController;
+import io.debezium.testing.system.tools.kafka.ConnectorConfigBuilder;
+import io.debezium.testing.system.tools.kafka.KafkaConnectController;
+import io.debezium.testing.system.tools.kafka.KafkaController;
+
+public class MysqlTestsWithDbOperations extends ConnectorTest {
```

Comment:
This class is not necessary. And also you should take more time thinking about naming. You created a class that is named Tests but there are no tests :) It is simply a utility class we should avoid as much as possible...